anyway/flask_app.py

```diff
@@ -398,7 +395,7 @@ def yishuv_symbol_to_name():
     logging.debug("getting yishuv symbol and yishuv name pairs")
     markers = (
         db.session.query(AccidentMarkerView.yishuv_name, AccidentMarkerView.yishuv_symbol)
-        .filter(not_(AccidentMarkerView.yishuv_name == None))
+        .filter(not_(AccidentMarkerView.yishuv_name is None))
```

Comment:
Notice it's not a pythonic operation but SqlAlchemy, from reading this message on Stackoverflow(below) i don't believe it will work

https://stackoverflow.com/a/5632224/13771969

Reply:
great catch, I think we are in higher version so maybe it's ok, but will revert anyway